Sharprompt/Prompt.Customize.cs

```diff
@@ -1,11 +1,13 @@
-﻿using System;
+using System;
 
 namespace Sharprompt
 {
     public static partial class Prompt
     {
         public static class ColorSchema
         {
+            public static ConsoleColor DoneSymbolColor { get; set; } = ConsoleColor.Green;
+            public static ConsoleColor PromptSymbolColor { get; set; } = ConsoleColor.Green;
```

Comment:
Remove the `Color` suffix.

```suggestion
            public static ConsoleColor DoneSymbol { get; set; } = ConsoleColor.Green;
            public static ConsoleColor PromptSymbol { get; set; } = ConsoleColor.Green;
```